frontend/src/app/gitImport.tsx

```diff
@@ -0,0 +1,159 @@
+                  <Alert
+                    variant="danger"
+                    title={this.state.submitMsg}
+                    action={<AlertActionCloseButton onClose={this.dismissNotification} />}
+                  />
+                </div>
+              )}
+
+              <FormGroup
+                label="Repository URL"
+                fieldId="repository-url"
+              >
+                <TextInput id="repository-url" type="text" placeholder="Repository" value={repository} onChange={this.handleRepoInput} />
+              </FormGroup>
+              <br />
+              <FormGroup
+                label="Branch"
+                fieldId="branch-name"
+              >
+                <TextInput id="branch-name" type="text" placeholder="Branch" value={branch} onChange={this.handleBranchInput} />
+              </FormGroup>
+              <br />
+              <Button aria-label="Submit the repository and branch information to the git integration service." onClick={this.cloneRepo}>Submit</Button>
+              <div>
+                {this.loginRedirect()}
+                {this.renderRedirect()}
+              </div>
+            </div>
+          </div>
+        </Bullseye>
+      </React.Fragment>
+    );
+  }
+
+  private handleRepoInput = repository => {
+    this.setState({ repository });
+    console.log("Repository " + repository)
+  };
+
+  private handleBranchInput = branch => {
+    this.setState({ branch });
+    console.log("Branch " + branch)
+  };
+
+  private cloneRepo = (postBody) => {
+    console.log("My repo is: " + this.state.repository + " and branch is " + this.state.branch + ".")
+    if (this.state.repository === "") {
+      this.setState({ isMissingFields: true })
+    } else {
+      const hdrs = {
+        'Accept': 'application/json',
+        'cache-control': 'no-cache'
+      }
+
+      fetch('/content/repositories/GIT_INTEGRATION_URL')
+        .then((resp) => resp.text()).then((text) => {
+          this.setState({ git2pantheonURL: text })
+        }).then(() => {
+          console.log("The git2pantheon URL is: " + this.state.git2pantheonURL)
+          const payload = {
+            branch: this.state.branch,
+            repo: this.state.repository
+          };
+          fetch(this.state.git2pantheonURL + "/clone", {
+            body: JSON.stringify(payload),
+            method: "POST"
+          }).then(response => {
+            if (response.status === 201 || response.status === 200) {
+              console.log(" Works " + response.status)
+              this.setState({ isFormSubmitted: true, isSucess: true, msgType: "success", submitMsg: "The git import has been submitted it might take up to 1 minute to see it in the module library" })
+            } else if (response.status === 500) {
+                console.log(" Failed " + response.status)
+                response.text().then((text) => {
+                  this.setState({ submitMsg: text })
```

Comment:
This needs a bit of polish. When I run this locally with no go component, I get an error dialog that fills the entire page and starts with `<html> <head> <title>Error while processing /&lt;!DOCTYPE html&gt;&lt;html...`

Reply:
mm that is unexpected. Let me see why my dialog box is not showing up.

Reply:
Thanks for catching this, the issue came from the configuration not being defined in sling and react was doing a call to localhost:8080/clone which redirects to /pantheon so the whole html payload was in the response. This should be fixed now.